mobile/library/java/io/envoyproxy/envoymobile/engine/EnvoyConfiguration.java

```diff
@@ -54,7 +54,7 @@ public enum TrustChainVerification {
   public final String appId;
   public final TrustChainVerification trustChainVerification;
   public final String virtualClusters;
-  public final List<EnvoyNativeFilterConfig> nativeFilterChain;
+  public final List<EnvoyNativeConfig> nativeFilterChain;
```

Comment:
Should the variable be renamed to match?

```suggestion
  public final List<EnvoyNativeConfig> nativeConfigs;
```

Reply:
Good question  this stays as is because it's filter chain configuration.  In the API change one we also switch stats from "big yaml blob" to "name and typed config" though now that I think about it I wonder if I can rework so I'm going to hold on this one
/wait